pkg/asset/images.go

```diff
@@ -3,11 +3,11 @@ package asset
 // DefaultImages are the defualt images bootkube components use.
 var DefaultImages = ImageVersions{
 	Etcd:            "quay.io/coreos/etcd:v3.3.12",
-	Flannel:         "quay.io/coreos/flannel:v0.10.0-amd64",
+	Flannel:         "quay.io/coreos/flannel:v0.11.0-amd64",
 	FlannelCNI:      "quay.io/coreos/flannel-cni:v0.3.0",
 	Calico:          "quay.io/calico/node:v3.0.3",
```

Comment:
Would it make sense to update calico too? We are using bootkube and calico, we just upgraded to calico 3.10.1 (https://docs.projectcalico.org/v3.10/release-notes/#v3.10.1)

We are templating this vars outside of bootkube (to keep bootkube on the bootkube functionality), but maybe it makes sense to update so it is clear that is supported and there is no reason to use an old calico version.

What do you think?

Reply:
well, we are still discussing how to enable multiple CNI's which is why i didnt want to update it at this time, but i can open an issue so that CNI's can be discussed there

Reply:
Not sure I follow. There are several CNIs already, isn't the plan to just keep them as they are at least until the way to have many is revisited? Why not change the version number, then? Am I missing something?

Also, it would be great if you can create an issue or document in some way what you mean with still discussing with how to enable different CNIs (like what are the problems you see now, what you would like in a solution, or what are you at in the current disucssion)

Reply:
Calico in its canal implementation is still fixed with the `experimental` flag, since the jenkins which ran the e2e tests are no longer present ( they were run by coreos ) we still have to figure out where/how to run the e2e `flannel-e2e`, `calico-e2e` & `conformance` needs to run, it is a part of the roadmap to get e2e running. If its just a version bump+config i guess without e2e, i can send a separate PR for it since its still experimental

Reply:
Oh, okay. Let me know if I can help :)